tuf/store/interfaces.go

```diff
@@ -12,7 +17,8 @@ type MetadataStore interface {
 
 // PublicKeyStore must be implemented by a key service
 type PublicKeyStore interface {
-	GetKey(role string) ([]byte, error)
+	GetKey(role string) (data.PublicKey, error)
+	RotateKey(role string, cs signed.CryptoService, roots ...data.PublicKey) (data.PublicKey, error)
```

Comment:
NM, sorry, just reread https://github.com/docker/notary/pull/553#discussion_r52268867, which says they should return `[]byte` :)  

Hmm... If we were using GRPC, would you still say that `GetKey` and `RotateKey` should return `[]bytes` and not `data.PublicKey`?  For instance I'd think that `GetMeta` and `SetMeta` should still take and return `[]bytes`, since that's what TUF operates on (to do checksumming, etc.)